src/Commands/Pages/GetPageLikedByInformation.cs

```diff
@@ -0,0 +1,52 @@
+﻿
+using PnP.Core.QueryModel;
+using PnP.PowerShell.Commands.Base.PipeBinds;
+using PnP.PowerShell.Commands.Model;
+using System;
+using System.Collections.Generic;
+using System.Management.Automation;
+using System.Text;
+
+namespace PnP.PowerShell.Commands.Pages
+{
+    [Cmdlet(VerbsCommon.Get, "PnPPageLikedByInformation")]
+    [OutputType(typeof(PageLikedByInformation))]
+    public class GetPageLikedByInformation : PnPWebCmdlet
+    {
+        [Parameter(Mandatory = true, ValueFromPipeline = true, Position = 0)]
+        public PagePipeBind Identity;
+
+        protected override void ExecuteCmdlet()
+        {
+            var clientSidePage = Identity.GetPage(Connection);
+
+            if (clientSidePage == null)
+                throw new Exception($"Page '{Identity?.Name}' does not exist");
+
+            var pageLikeInformation = clientSidePage.GetLikedByInformationAsync().GetAwaiter().GetResult();
+            
+            var likes = pageLikeInformation.LikedBy.AsRequested();
+
+            var likesList = new List<PageLikedByInformation>();
+            
+            if(likes != null)
+            {
+                foreach( var liked in likes)
+                {
+                    var likedInfo = new PageLikedByInformation
+                    {
+                        Name = liked.Name,
+                        Mail = liked.Mail,
+                        LoginName = liked.LoginName,
+                        Id = liked.Id,
+                        CreationDate = liked.CreationDate
+                    };
+
+                    likesList.Add(likedInfo);
+                }
```

Comment:
Do we need a foreach for this ? I guess the data retrieved from the API and PnP Core would also have a foreach loop , maybe we can simply use it like 

```
var likes = pageLikeInformation.LikedBy.AsRequested();
WriteObject(likes, true);
```

Thoughts ?

Reply:
Hi @gautamdsheth, the [PnP.Core](https://pnp.github.io/pnpcore/using-the-sdk/pages-social.html#enumerating-page-likes) returns the  `likes` as Enumerable, which we definetly need to iterate (as same in `Get-PnPListItemComment` command). 

Instead of the example given in the PnP.Core as below, 

```
// Get a list of users who liked this page
var pageLikeInformation = await newPage.GetLikedByInformationAsync();

// Was page liked by the current user?
bool pageLikedByCurrentUser = pageLikeInformation.IsLikedByUser;

// Enumerate the persons that liked this page
foreach(var likedByUser in pageLikeInformation.LikedBy.AsRequested())
{
    // do something with the information about the user who liked this page
}
```

I have added an extra check to see if the likes are null, hence as the below code, 

```
var pageLikeInformation = clientSidePage.GetLikedByInformationAsync().GetAwaiter().GetResult();
var likes = pageLikeInformation.LikedBy.AsRequested();

if(likes != null)
{
    foreach( var liked in likes)
    {
        var likedInfo = new PageLikedByInformation
        {
            Name = liked.Name,
            Mail = liked.Mail,
            LoginName = liked.LoginName,
            Id = liked.Id,
            CreationDate = liked.CreationDate
        };

        likesList.Add(likedInfo);
    }
}
```


Thanks,
Nishkalank Bezawada

Reply:
Yeah, you are right, we need this !

Also, for the future PRs, please don't add personal identifiers in the docs, lets keep it neutral like user1 or test or something like that.

Have updated the docs here, so all good :)